### PR TITLE
Update ProductAdmin.php

### PR DESCRIPTION
**Pull request by chusket**
fix Compile Error: Declaration of Sonata\ProductBundle\Admin\ProductAdmin::validate() must be compatible with Sonata\AdminBundle\Admin\AdminInterface::validate(Sonata\AdminBundle\Validator\ErrorElement $errorElement, $object)

### CODE REVIEW

**Comment by OskarStark** [comment]
this is wrong IMO

ping @rande

**Comment by chusket** [comment]
Hi OskarStark!,  Could you argue why?

File: src/ProductBundle/Admin/ProductAdmin.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
 use Sonata\Component\Currency\CurrencyDetectorInterface;
 use Sonata\Component\Product\Pool;
 use Sonata\Component\Product\ProductInterface;
-use Sonata\CoreBundle\Validator\ErrorElement;
+use Sonata\AdminBundle\Validator\ErrorElement;
```
Comment (OskarStark) [inline src/ProductBundle/Admin/ProductAdmin.php]: i meant this change

**Comment by OskarStark** [comment]
ping @chusket

**Pull request closed without merging**